app/gui2/src/components/GraphEditor/GraphEdge.vue

```diff
@@ -442,15 +452,21 @@ const arrowTransform = computed(() => {
       @pointerenter="hovered = true"
       @pointerleave="hovered = false"
     />
-    <path ref="base" :d="basePath" class="edge visible" :style="baseStyle" />
+    <path
+      ref="base"
```

Comment:
This path should have a `v-if`, as we don't want to display two paths for each connection. We want to pay the cost of rendering two layers only for currently active edge (i.e. one extra path, not hundreds). Then, to not complicate things, the `ref="base"` could be on the always-visible front layer. It is only used for `getTotalLength` call, and the result will be identical for both (since it is the same path).